### PR TITLE
ci: added default shell for ockam-package workflow

### PR DESCRIPTION
**Pull request by rghdrizzle**
This issue solves a problem in https://github.com/build-trust/ockam/issues/4001 ,the problem was , in every job or run command there was a need for mentioning which shell do we need to run the command in .

To solve this issue , I used a default to create a map of the settings that will apply to every job in the workflow and gave a default value for shell as bash since every run command in the workflow runs in bash

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: .github/workflows/ockam-package.yml [hunk 1]
```diff
@@ -24,6 +24,10 @@ env:
   ARTIFACT_NAME: ockam
   ORGANIZATION: ${{ github.repository_owner }}
 
+defaults:
+  run:
+    shell: bash
```
Comment (metaclips) [inline .github/workflows/ockam-package.yml]: Lint CI fails due to formatting https://github.com/build-trust/ockam/actions/runs/4432846749/jobs/7777477628?pr=4471 can we remove the whitespace here?

**Review by metaclips — CHANGES_REQUESTED** [review]
Close to a merge, let's make CI happy

File: .github/workflows/ockam-package.yml (hunk 1 above)
Comment (rghdrizzle) [reply .github/workflows/ockam-package.yml]: Ok I removed the whitespace in 29:17

**Comment by metaclips** [comment]
@rghdrizzle can you squash your commits

**Comment by rghdrizzle** [comment]
@metaclips done squashing the commits

**Comment by metaclips** [comment]
Seems we didn't sign our commit when we squashed it, can you run 
```bash
git commit --amend -S -m "ci: added default shell for ockam-package workflow"
```

**Comment by rghdrizzle** [comment]
@metaclips there seems to be some issues with my signing for some reason at the moment , shall I open a new pr ? or fix my signing issue in this pr ?

**Comment by metaclips** [comment]
> @metaclips there seems to be some issues with my signing for some reason at the moment , shall I open a new pr ? or fix my signing issue in this pr ?

Any is fine with me 🙂

**Comment by rghdrizzle** [comment]
Ok I'll close this pr and open a new one , sorry for the inconvenience :)

**Pull request closed without merging**